core/src/wallet/algorand/AlgorandBlockchainExplorer.cpp

```diff
@@ -57,6 +56,8 @@ namespace algorand {
         const std::string minRoundQueryParam = "{}&min-round={}";
         const std::string maxRoundQueryParam = "{}&max-round={}";
 
+        const std::string apiToken = "51QbkNgumz6aMzgl7qL2FabvZsREiCyw6VLcuNCj";
```

Comment:
Secret shouldn't be hardcoded. Put that value in a environment variable for security and scalability

Reply:
I definitely agree.
I think the way to go is by using a configuration key, similar to the ones defined in `api::Configuration`, however do you know if there's a way to use an environment variable to set the value of such key?... In particular for libcore's tests.

Reply:
You can use `std::getenv` to get the API token however we have to add an entry point to call - as a init function - before using the specific configuration key. This can be done when the wallet is initialized or somewhere else more appropriate; if any.

Reply:
I put it in `algorand::WalletFactory::build`.